Restore broken folder credentials icon

NotMyFault opened the pull request:
`folder-store.svg` was moved from the credentials plugin to the folder plugin, the old reference displays a broken image:
![Screenshot 2022-07-25 at 18 07 48](https://user-images.githubusercontent.com/13383509/180825329-160eadd9-fb30-4c26-ad91-4737449e6cef.png)
The change proposed addresses the issue and restores the icon:
![Screenshot 2022-07-25 at 18 09 33](https://user-images.githubusercontent.com/13383509/180825450-85226572-6dee-4dfa-9314-4867c7dd654f.png)

jglick commented:
Is this a regression caused by some identifiable previous PR? If so, please paste a link to that for reference purposes

NotMyFault commented:
Added by https://github.com/jenkinsci/cloudbees-folder-plugin/pull/239/files to the folders plugin and removed by https://github.com/jenkinsci/credentials-plugin/pull/321 from the credentials plugin.

jglick commented:
(please paste **Conversation** links only)

NotMyFault commented:
Just submitted https://github.com/jenkinsci/credentials-plugin/pull/346 as follow up.

jglick commented:
@alecharp did you intentionally disable automerge here?

alecharp commented:
This was not intentional. Sorry for that.
